### ISSUE TITLE
List of problematic words

### ISSUE THREAD

**Issue by ampli** [opencog/link-grammar#1289]
Many times the role of missing words can be correctly inferred by the UNKNOWN-WORD device.
However, sometimes the dict defines a word partially (missing POS), or an unknown word is misclassified by regex or gets spell-corrected.

I also have an additional list of words that are totally missing (some of them may have similar problems).

Missing POS
----
affection n
architect v
archive n
continue n
debug n
disaffection n
discarding n
disjunct n
fallback n
hash n
hashtag v
inverse v
merge n
negative n
optional n
outgoing n
padding n
positive n ("false positive" doesn't parse)
postprocessing n-u
predate n
prehistory n
preset a
pruning n
redistribution n
running n
speedup n-u
tar n
tombstone v
trim n

Misclassified by regex
----
hyperfine a (\<MC-NOUN-WORDS>.n)
iterable n (\<ADJ-WORDS>.a)
misdefine v (\<MC-NOUN-WORDS>.n)
multiline a (\<MC-NOUN-WORDS>.n)
preprovide v (\<MC-NOUN-WORDS>.n)
reenable v (\<ADJ-WORDS>.a)
untar v (\<ADJ-WORDS>.a)
versioning n (\<ING-WORDS>.g)

Missing plural noun words are misclassified by regex
----
indirections n (\<S-WORDS>.v)
(There are more.)

Spell corrected (partial list)
----
bitfield n
changelog n
overlink v
toolset n

Idioms
------
flounce out (corpus-failures.batch: And she flounces out.)
make file n

Ref: Comments in PR #1284.

**Comment by linas** [comment]
Trying to fix these now.
 
negative.n  was already in the dict.
tar.n-u and padding.n-u was already in the dict as a mass noun

Do you have example failing sentences for these?

**Comment by linas** [comment]
I made the following verbs transitive.  Perhaps they should be optionally transitive?
```
architect
hashtag
inverse
misdefine
overlink
reenable
tombstone
```
They're transitive, because you can say "I will architect it" but its awkward to say "I will architect"

Do you have any examples where these are intranstive?

**Comment by ampli** [comment]
tar:
The word `tar.s` got added on Apr 30, 2021, and it seems it happened after I have added `tar.n` to my list.
It was mentioned as a kind of substance, and I noticed Wiktionary says it can be countable.

padding:
I don't have the sentence now, but it included the word [`paddings`](https://en.wiktionary.org/wiki/paddings).

(I will add dates and examples to my word list from now on.)

BTW, I first wrote "It got added as ..." and noted `got` doesn't parse.
